RetailCoder.VBE/Root/RubberduckModule.cs

```diff
@@ -61,6 +62,8 @@ public override void Load()
             _kernel.Bind<AddIn>().ToConstant(_addin);
             _kernel.Bind<RubberduckParserState>().ToSelf().InSingletonScope();
             _kernel.Bind<GitProvider>().ToSelf().InSingletonScope();
+            _kernel.Bind<NewUnitTestModuleCommand>().ToSelf().InSingletonScope();
+            _kernel.Bind<NewTestMethodCommand>().ToSelf().InSingletonScope();
```

Comment:
we need to have a [technical-debt] issue created for this. the command bindings should be automatically done with the convention bindings.